core/client/src/transaction_builder/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Utility struct to build a transaction.
+
+#[cfg(feature = "std")]
+mod transaction_builder;
+#[cfg(feature = "std")]
+pub use transaction_builder::*;
```

Comment:
could just put a `#![cfg(feature = "std")]` at the top of the file.

Reply:
(but that's probably not the desired behavior -- don't we want the `TransactionBuilder` API to be exposed in `no_std`? )